readium/lcp/src/main/java/org/readium/r2/lcp/LcpPublicationRetriever.kt

```diff
@@ -183,27 +201,23 @@ public class LcpPublicationRetriever(
             download: DownloadManager.Download
         ) {
             coroutineScope.launch {
-                val lcpRequestId = RequestId(requestId.value)
+                val data = downloadsRepository.getDownload(requestId)
+                    ?: return@launch // Repository is corrupted.
```

Comment:
We don't need to report this error to the listener?

(same for other occurrences below)

Reply:
I think it could be interesting to inform the user in that case. But to do this, we would need to add a member to `LcpError` hierarchy or design a specific one, which I was not willing to do.

Reply:
Worst case, we have `LcpError.Runtime` and `LcpError.Unknown`. I'm concerned that the app would get stuck on this download if no error is reported there, what would happen?

Reply:
Oh, no, I know why I did that. You have no choice because you can't get the request ID to pick up the listeners to call.

Reply:
Ha yeah, makes sense.